### ISSUE TITLE
`UploadTo` utils break when the original filename does not have an extension

### ISSUE THREAD

**Issue by syphar** [codingjoe/django-stdimage#61]
https://github.com/codingjoe/django-stdimage/blob/master/stdimage/utils.py#L17
In this code part
```python
def __call__(self, instance, filename):
    defaults = {
        'ext': filename.rsplit('.', 1)[1],
        'name': filename.rsplit('.', 1)[0],
        'path': filename.rsplit('/', 1)[0],
        'class_name': instance.__class__.__name__,
    }
    defaults.update(self.kwargs)
    return os.path.join(self.path_pattern % defaults, self.file_pattern % defaults).lower()
```

I'm quite certain we should replace the `.rsplit` usages with `os.path.split` and `os.path.splitext`

**Comment by codingjoe** [comment]
good point, we already changed that for the field in 2c3eef66fcf5060f96e2b0eca879f7c275f17859

**Comment by codingjoe** [comment]
Released in 2.1.0. Please not that the `file_pattern` no longer contains a dot. Rather was the extension separator moved into the extension as suggested python `os.path`
